docs/faq.md

```diff
@@ -0,0 +1,252 @@
+
+### <a name="additional-metadata"></a> How can I specify additional tuning metadata?
+
+By default, Syne Tune stores the time, the names and modes of the metric being tuner, the name of the entrypoint, the name backend and the scheduler name.
+You can also add custom metadata to your tuning job by setting `metadata` in `Tuner` as follow:
+
+```
+tuner = Tuner(
+    ...
+    tuner_name="plot-results-demo",
+    metadata={"tag": "special-tag", "user": "alice"},
+)
+```
+
+All Syne Tune and user metadata are saved when the tuner starts under `metadata.json`.
+
+
+### <a name="logging-additional-information"></a> How do I append additional information to the results which are stored? 
+
+Results are processed and stored by callbacks passed to `Tuner`, in particular see https://github.com/awslabs/syne-tune/blob/main/syne_tune/tuner_callback.py#L80. In order to add more information to these results, you can inherit from `StoreResultsCallback`. A good example is given in https://github.com/awslabs/syne-tune/blob/main/syne_tune/optimizer/schedulers/searchers/searcher_callback.py#L63.
+
+If you run experiments with tabulated benchmarks using the `SimulatorBackend`, as demonstrated in https://github.com/awslabs/syne-tune/blob/main/examples/launch_nasbench201_simulated.py, results are stored by `SimulatorCallback` instead, and you need to inherit from this class, as shown in https://github.com/awslabs/syne-tune/blob/main/syne_tune/optimizer/schedulers/searchers/searcher_callback.py#L88.
+
+### <a name="remote-tuning"></a> I don’t want to wait, how can I launch the tuning on a remote machine?
+
+You can use the remote launcher to launch a tuning on a remote machine. The remote launcher supports both `LocalBackend` and `SagemakerBackend`. In the former case, multiple trials will be evaluated on the remote machine (one use-case being to use a beefy machine), in the latter case trials will be evaluated as separate SageMaker training jobs.
+
+See for an example on how to run tuning with the remote launcher: https://github.com/awslabs/syne-tune/blob/main/examples/launch_height_sagemaker_remotely.py
+
+### <a name="experiment-parallel"></a> How can I run many experiments in parallel?
+
+You can call the remote launcher multiple times to schedules a list of experiments. In some case, you will want more flexibility and directly write your experiment loop, you can check https://github.com/awslabs/syne-tune/tree/main/benchmarking/benchmark_loop for an example.
+
+### <a name="results-remote-tuning"></a> How can I access results after tuning remotely?
+
+You can either call `load_experiment("{tuner-name}")` which will download files from s3 if the experiment is not found locally. You can also sync directly files from s3 under `~/syne-tune/` folder in batch for instance by running:
+
+```
+aws s3 sync s3://{sagemaker-default-bucket}/syne-tune/{tuner-name}/ ~/syne-tune/
```

Comment:
This will be tedious when you ran lots of experiments, maybe random repeats etc. If experiment-name is used, I have some instructions in README.txt of ExperimentToolkit how to do a bulk download and also filter out things you do not want (e.g., all the logs and checkpoints, the dill files, etc.).

Happy to add this here.

Reply:
I updated with a way to download only the desired files.